root/layout/components/Footer.js

```diff
@@ -9,87 +9,568 @@
-        <a className="internal" href={DONATE_URL}>{l('Donate')}</a>
-        <a className="internal" href="//wiki.musicbrainz.org/">{l('Wiki')}</a>
-        <a className="internal" href="https://community.metabrainz.org/">{l('Forums')}</a>
-        <a className="internal" href="/doc/Communication/IRC">
-          {l('Chat (IRC)')}
-        </a>
-        <a className="internal" href="http://tickets.metabrainz.org/">{l('Bug Tracker')}</a>
-        <a className="internal" href="https://blog.metabrainz.org/">{l('Blog')}</a>
-        <a className="internal" href="https://twitter.com/MusicBrainz">{l('Twitter')}</a>
-
-        {DBDefs.BETA_REDIRECT_HOSTNAME ? (
-          <a
-            className="internal"
-            href={
-              '/set-beta-preference?' + returnToCurrentPage($c)
-            }
-          >
-            {DBDefs.IS_BETA
-              ? l('Stop using beta site')
-              : l('Use beta site')}
-          </a>
-        ) : null}
-      </p>
-
-      <p className="right">
-        {exp.l(
-          `Brought to you by {MeB|MetaBrainz Foundation} and our
-           {spon|sponsors} and {supp|supporters}. Cover Art provided
-           by the {caa|Cover Art Archive}.`,
-          {
-            caa: '//coverartarchive.org/',
-            MeB: 'https://metabrainz.org/',
-            spon: 'https://metabrainz.org/sponsors',
-            supp: 'https://metabrainz.org/supporters',
-          },
-        )}
-
-        {DBDefs.DB_STAGING_SERVER && DBDefs.GIT_BRANCH ? (
-          <>
+    <section className="footer">
+      <div className="ms-4 me-4 mb-3 ps-4 pb-3 pe-4 border-top container">
+        <div className="row mb-4 mt-2 pt-2">
+          <div className="col-sm-12 col-md-4">
+            <h3>
+              <img
+                alt="MusicBrainz"
+                src="../../static/images/meb-logos/musicbrainz.svg"
+                width="180"
+              />
+            </h3>
             <br />
-            {exp.l('Running: {git_details}', {
-              git_details: (
-                <span
-                  className="tooltip"
-                  key="git_details"
-                  title={DBDefs.GIT_MSG}
-                >
-                  {DBDefs.GIT_BRANCH}
-                  {' '}
-                  {bracketedText(DBDefs.GIT_SHA)}
+            <p className="fs-4">
+              {l(`MusicBrainz is an open music encyclopedia that collects
+              music metadata and makes it available to the public.`)}
+            </p>
+            <ul className="list-unstyled">
+              <li>
+                <span className="fs-4">
+                  {l('Development IRC: ')}
+                </span>
+                <a
+                  className="fw-bold fs-4"
+                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('#metabrainz')}
+                </a>
```

Comment:
`#musicbrainz` is the more general channel for the project, though I think it makes sense to just link to our IRC documentation, as the existing footer does, since that lists all of our channels and has other useful information. That would also allow people use their own IRC client.

Reply:
Right, added both the IRC channels and tweaked the UI a bit!

Reply:
Thanks, but that wasn't my suggestion. :)  By "link to our IRC documentation, as the existing footer does," I meant https://musicbrainz.org/doc/Communication/IRC.  This contains useful information and lets people connect with their own client.  If I was looking to join a project's IRC channel, I wouldn't want a direct link to some web client: I'd want to read about what the purpose of each channel is first, and then connect with my own client.